internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -236,3 +236,24 @@ Resources:
                 Resource: !Sub arn:aws:s3:::${Bucket}/*
       ManagedPolicyArns:
         - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+  
+  EnvManagerS3Access:
+    Metadata:
+      aws:copilot:description: A policy that gives the Env Manager role access to this site's S3 Bucket
+    Type: AWS::IAM::Policy
```

Comment:
What on earth! So this is adding an inline policy to the env manager role? I didn't know we could do this 🤯